linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls_cacert/rule.yml

```diff
@@ -10,6 +10,7 @@ description: |-
     using correct path for the <tt>DefaultNetstreamDriverCAFile</tt>
     global option in <tt>/etc/rsyslog.conf</tt>, for example
     <pre>global(DefaultNetstreamDriverCAFile="/etc/pki/tls/cert.pem") *.*</pre>
+    Replace the <tt>/etc/pki/tls/cert.pem</tt> in the configuration line above with the path to the file with CA certificate generated for the purpose of remote logging.
```

Comment:
Same as in the other rule, please just add a command to configure the rule.

Reply:
Wouldn't be the exact command more suitable for ocil?

Reply:
In the OCIL the command is for checking, here is to configure.